lib/guest_installation_and_configuration_base.pm

```diff
@@ -1504,7 +1504,18 @@ by passing non-empty arguments using hash.Call config_guest_installation_media t
 set correct installation media. For directkernel installation method which uses
 virt-install --install, please also refer to following link for more information
 https://manpages.opensuse.org/Tumbleweed/virt-install/virt-install.1.en.html
-
+For 'location' installation method, virt-install command looks like:
+--location [guest_installation_media],[guest_installation_method_others]
+--extra-args root=live:[guest_installation_fine_grained_media]
+--extra-args inst.install_url=[guest_installation_fine_grained_repos]
+--extra-args [guest_installation_extra_args] 
+For 'directkernel' installation method, virt-install command looks like:
+--install kernel=[guest_installation_fine_grained_media]/linux,
+initrd=[guest_installation_fine_grained_media]/initrd,
+kernel_args=root=live:[guest_installation_media],inst.install_url=
```

Comment:
This "root=live:" is specific for sle16 behavior, right? From below code, it seems a general way, regardless of product versions, for both installation methods of 'location' and 'direct kernel'...

Reply:
Yes. Just use "location" without specifying guest_installation_fine_grained_media and guest_installation_fine_grained_repos. SLE15 does not need these parameters.